iothub/device/src/Transport/AmqpIoT/AmqpIoTSession.cs

```diff
@@ -226,6 +227,11 @@ TimeSpan timeout
                 amqpLinkSettings.AddProperty(AmqpIoTErrorAdapter.ChannelCorrelationId, CorrelationId);
             }
 
+            if (!deviceIdentity.AmqpTransportSettings.AuthenticationChain.IsNullOrWhiteSpace())
+            {
+                amqpLinkSettings.AddProperty("AuthChain", deviceIdentity.AmqpTransportSettings.AuthenticationChain);
```

Comment:
Move "AuthChain" to a constant. There is a constants file maintained here: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/device/src/Transport/AmqpIoT/AmqpIoTConstants.cs

Reply:
in Iothub amqp properties names have the format: AmqpConstants.Vendor + ":api-version", where AmqpConstants.Vendor = "com.microsoft". I suggest to use the same format  mqpConstants.Vendor + ":auth-chain"

Reply:
I agree, this is better @ancaantochi !